src/ids-host.test.ts

```diff
@@ -25,6 +25,62 @@ async function mkPromise<T>(lookup: () => T): Promise<T> {
   return lookup();
 }
 
+describe("isUrlSubjectToDynamicUrls", () => {
+  type TestCase = {
+    inputUrl: string;
+    inScope: boolean;
+  };
+
+  const testCases: TestCase[] = [
+    {
+      inputUrl: "https://example.com/bar",
+      inScope: false,
+    },
+    {
+      inputUrl: "https://install.determinate.systems/aoeu",
+      inScope: true,
+    },
+    {
+      inputUrl: "https://install.determinate.systems/",
+      inScope: true,
+    },
+    {
+      inputUrl: "https://install.determinate.systems",
+      inScope: true,
+    },
+    {
+      inputUrl: "https://install.determinate.systems:123/",
+      inScope: false,
+    },
+    {
+      inputUrl: "http://install.determinate.systems/",
+      inScope: false,
+    },
+    {
+      inputUrl: "https://install.detsys.dev/",
+      inScope: false, // FALSE: only subdomains are valid
```

Comment:
Why is this false, if `install.determinate.systems` isn't false...?

Reply:
``
const DEFAULT_IDS_HOST = "https://install.determinate.systems";
```

```
  isUrlSubjectToDynamicUrls(url: URL): boolean {
    if (url.origin === DEFAULT_IDS_HOST) {
      return true;
    }
```

but we could make this one truthy if you think that is a good idea

Reply:
Hm, OK. I don't know how I feel about that, but I don't have an easy suggestion on how to fix that. It's probably OK.